README.md

```diff
@@ -9,36 +9,50 @@ A juju charm deploying and managing Discourse on Kubernetes. Discourse is the
 100% open source discussion platform. Use it as a mailing list, discussion
 forum or long-form chat room.
 
-This charm simplifies initial deployment and "day N" operations of Discourse
-on Kubernetes, such as scaling the number of instances, integration with SSO,
-access to S3 for redundant file storage and more. It allows for deployment on
-many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) or
+Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling, and more.
+For Charmed Discourse, this includes:
+  - Scaling
+  - Integration with SSO
+  - Integration with S3 for redundant file storage
+
+It allows for deployment on
+many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
 [Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
 offerings.
 
-As such, the charm makes it easy for those looking to take control of their own
-discussion platform whilst keeping operations simple, and gives them the
-freedom to deploy on the Kubernetes platform of their choice.
+For information about how to deploy, integrate, and manage this charm,
```

Comment:
Nitpick (to adhere to style guide): Remove comma between after "integrate", in other words:
> For information about how to deploy, integrate and manage this charm...